### DISCUSSION TITLE
Questions about the "aodReduced" GOCO06s used in the template

### DISCUSSION THREAD

**Discussion by ReubenLi1999** [groops-devs/groops#236] [Q&A] [answered]
Hi developers,

### Description

I have encountered some confusion regarding the "aodReduced" GOCO06s gravity field used in the 020groopsVariational.xml file from the provided [template](https://groops-devs.github.io/groops/html/cookbook.gravityFieldGrace.html). According to your configuration (see the figure below), the gravity field used in the "forces" is "timeSplines_background_10min.dat" generated by the "aodReduced" GOCO06s field (static + trend + seasonal) and the AOD1B time splines in the "Gravityfield2Timesplines" program. 

However, I am uncertain about the term "aodReduced" in this context, especially considering that the AOD1B product is also integrated into the program. Additionally, I have noticed some difference between the coefficients of the regular GOCO06s and the "aodReduced" version.

### Questions

Could you please kindly provide clarification on the following points:

1. The definition and significance of the "aodReduced" GOCO06s gravity field.
2. The distinctions between the regular and "aodReduced" versions of GOCO06s.
3. Guidance on which type of GOCO06s (regular or aodReduced) to use when applying the original AOD1B file instead of your AOD1B time splines.

### Attachment

![image](https://github.com/groops-devs/groops/assets/49586424/e1552322-4633-470e-afc1-f234fe5629c4)

Thank you sincerely for your time and patience in addressing these inquiries.

Best regards,

Haosi

**Comment by tmayerguerr** [comment, marked as answer]
Hi Haosi,

- The file `AOD1B_RL06_dealiasing_timeSplines3h_n180_{timeStart:%y_%m}.dat`
consists of the original AOD1B model converted with [GraceAod2TimeSplines](https://groops-devs.github.io/groops/html/GraceAod2TimeSplines.html).

- For the orbit integration the force function is evaluated along the orbit at every position every 5 second. This can be quite time consuming. The spherical harmonic coefficients of most of these force models change hardly within 5 second. Therefore we evaluate the coefficients only every 10 minutes before, add them together and store the time series in the file `timeSplines_background_10min.dat` with [Gravityfield2TimeSplines](https://groops-devs.github.io/groops/html/Gravityfield2TimeSplines.html). For the orbit integration the coefficients are linearly interpolated to the orbit epoch.

- The published GOCO06s model describes the full static, trend and annual potential of the Earth including atmospheric and ocean masses. If you add AOD1B to consider the high frequent variations of atmosphere and ocean you have a double consideration of trend and annual signal. We estimated the trend and annual signal from the AOD1B time series and removed it from GOCO06s (GOCO06s_AodReduced = GOCO06s - AOD1B(static,trend,annual)).

best regards
Torsten

**Reply by ReubenLi1999** [reply]
Thank you very much indeed. Torsten. 

Your clarification is very clear and your second explaination may be a potential time-saver for me, because I am evaluating the force functions every one second now. I'm eager to conduct tests on this "time splines" method myself to witness its efficiency firsthand.

Thanks again for your invaluable insights.

Best regards,

 Haosi
